spec/setup_test_database.rb

```diff
@@ -7,5 +7,5 @@ def setup_test_database
 
 def add_row_to_test_database
   connection = PG.connect(dbname: 'chitter_test')
-  connection.exec("INSERT INTO peeps (message) values ('This is a peep!');")
+  connection.exec("INSERT INTO peeps (message, date) values ('This is a peep!', '1997');")
```

Comment:
Nice! Did you make a database migration to change the structure of the table? It's good practise to include that script in the migrations folder so that another dev can reproduce the database.